website/docs/r/storage_bucket.markdown

```diff
@@ -101,6 +125,22 @@ The `CORS` object supports the following:
 
 * `max_age_seconds` (Optional) Specifies time in seconds that browser can cache the response for a preflight request.
 
+* `server_side_encryption_configuration` (Optional) A configuration of server-side encryption configuration (documented below)
+
+The `server_side_encryption_configuration` object supports the following:
+
+* `rule` (Required) A single object for server-side encryption by default configuration. (documented below)
+
+The `rule` object supports the following:
+
+* `apply_server_side_encryption_by_default` (Required) A single object for setting server-side encryption by default. (documented below)
```

Comment:
Is it possible to enable encryption by default and somehow disable it? If not, what's the meaning of "by default" here?

Reply:
Name is inherited from compatibility requirements for S3 API. This configuration sets encryption configuration for bucket that will apply by _default_ for all new objects uploaded to bucket.